### DISCUSSION TITLE
Keyboard

### DISCUSSION THREAD

**Discussion by SmeronMaks** [ClassicOldSong/moonlight-android#42] [Q&A]
I came here from the original Moonlight and I don’t understand how to turn the keyboard into multi-touch mode, before in the original Moonlight this was done by pressing three fingers on the screen, but how can I do it here?

**Comment by ClassicOldSong** [comment]
Back Menu -> Show soft keyboard

**Reply by ClassicOldSong** [reply]
If you mean close the keyboard, press back or swipe from the edge on your phone.

**Reply by SmeronMaks** [reply]
Unfortunately, it is inconvenient, because the keyboard on the phone occupies more than half of the screen, and when you gesture back opens or the menu with the weather on Windows or the notification menu, you can not open the keyboard in another way? For example, pressing several fingers?

**Reply by SmeronMaks** [reply]
I'm writing through a translator, I apologize if it's not clear what I mean. It's just that if I open the keyboard through the back menu, I can't select the text input field on the computer, because the keyboard takes up most of the screen, and when I gesture back, the place where I was going to type is no longer active.

**Reply by ClassicOldSong** [reply]
Swipe form the position where the keyboard displays should not trigger the Windows thing. Or just swipe a little bit more carefully.

It is suggested to use Trackpad(natural) mode if you're on a phone, that would make cursor movement more usable with small screens.

**Reply by ClassicOldSong** [reply]
You can also run this command in PowerShell with Administrator privilege to remove the sidebar completely:

```pwsh
Get-AppxPackage *WebExperience* | Remove-AppxPackage
```
